test/common.js

```diff
@@ -112,6 +120,34 @@ class TestTools {
 		return new Sim.Battle(battleOptions);
 	}
 
+	/**
+	 * Creates a new Battle and returns it.
+	 *
+	 * @param {Object} [options]
+	 * @param {Team[]} [teams]
+	 * @returns {TestContext} Test context.
+	 */
+	testCtx(options, teams) {
```

Comment:
```suggestion
	testCtx(options, teams) {
```
Nit: `testContext` instead of `testCtx` for the function name is clearer and more consistent with the rest of the codebase. I'd lean towards `getTestContext` given it's a function, though.